configure.ac

```diff
@@ -728,7 +728,7 @@ AC_CHECK_LIB([gmp],[[__gmpn_sub_n]],GMP_LIBS=-lgmp, [AC_MSG_ERROR(libgmp missing
 AC_CHECK_HEADER([gmpxx.h],,AC_MSG_ERROR(libgmpxx headers missing))
 AC_CHECK_LIB([gmpxx],[main],GMPXX_LIBS=-lgmpxx, [AC_MSG_ERROR(libgmpxx missing)])
 
-RUST_LIBS="-lrustzcash"
+RUST_LIBS="-lrustzcash -ldl"
```

Comment:
I guess we need this if there are undefined symbols... but I don't see "-ldl" elsewhere in the 1.1.0 source yet zcashd is already statically linked with libdl:
```
 ldd bin/zcashd
	linux-vdso.so.1 =>  (0x00007fffd618f000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fcf33f81000)
	libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fcf33bff000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fcf339fb000)
	libanl.so.1 => /lib/x86_64-linux-gnu/libanl.so.1 (0x00007fcf337f7000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fcf335ef000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fcf332e6000)
	libgomp.so.1 => /usr/lib/x86_64-linux-gnu/libgomp.so.1 (0x00007fcf330c4000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fcf32eae000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fcf32ae4000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fcf34dc4000)
```

Reply:
`-ldl` is in the linker command [I posted above from CI](https://github.com/zcash/zcash/pull/3169#issuecomment-380984858).